core/src/toga/widgets/base.py

```diff
@@ -229,8 +229,10 @@ def window(self, window):
 
     @property
     def enabled(self):
-        """Is the widget currently enabled? i.e., can the user interact with the
-        widget?"""
+        """Is the widget currently enabled?
+
+        i.e., can the user interact with the widget?
```

Comment:
Breaking this sentence up doesn't make much sense; not sure why it's been broken here, but not in other identical sentences on other widgets.

Reply:
The first line of a docstring should be one complete sentence. Subsequent sentences will be separated by an empty line....as least AFAIK....

Reply:
Sure - that's my understanding as well... but it *didn't* do this on Box (and a couple of others). It's the inconsistency that is weirding me out.